build.gradle

```diff
@@ -493,6 +493,7 @@ def exportedProjects= [
         ":rest:rest-client",
         ":rest:rest-smartproxy",
         ":rm:rm-client",
+        ":rm:rm-server"
```

Comment:
What will be the impact to add rm-server here ?

Reply:
this is to export rm server javadoc only, it does not create any cross-project dependencies